main/codesig/test/cases/callgraph/realistic/3-par-merge-sort/src/Hello.scala

```diff
@@ -89,4 +90,4 @@ object Main{
         "hello.Main$#mergeSortSequential(scala.collection.immutable.IndexedSeq,scala.math.Ordering)scala.collection.immutable.IndexedSeq"
     ]
 }
-*/
+ */
```

Comment:
This inserted whitespace could be the issue of the test failure.

Reply:
```suggestion
*/
```

Reply:
Ah, looks like the IntelliJ code formatter has automatically formatted the file...
Your change seems to have fixed it 👍